mono/metadata/Makefile.am

```diff
@@ -176,7 +176,7 @@ libmono_icall_table_la_SOURCES = \
 libmono_icall_table_la_CFLAGS = $(SGEN_DEFINES) @CXX_ADD_CFLAGS@
 libmono_icall_table_la_LDFLAGS = $(libmonoldflags)
 if BITCODE
-libmono_icall_table_la_LIBADD = $(glib_libs) ../utils/libmonoutils.la ../sgen/libmonosgen.la libmonoruntimesgen.la
+libmono_icall_table_la_LIBADD =
 endif
```

Comment:
These were added for the other bitcode config (watchos probably). So it should be:
if BITCODE
if WASM
..
else
...
endif
endif